src/PrestaShopBundle/Resources/config/services/core/grid/grid_factory.yml

```diff
@@ -124,6 +124,15 @@ services:
         calls:
           - [setHookDispatcher, ['@prestashop.core.hook.dispatcher']]
 
+    prestashop.core.grid.factory.currency_for_language:
```

Comment:
We dont need this anymore 😉(code dedicated to test the behavior)

Reply:
removed